### PR TITLE
Elasticsearch/database search fallback behaviour (#5957)

### PR DESCRIPTION
**Pull request by hmmmmmmm**
#### Summary
I updated `database-configuration-settings.rst` to include requested details about Elasticsearch, Bleve and database search fallback behaviour.

#### Ticket Link
https://github.com/mattermost/docs/issues/5957

### CODE REVIEW

File: source/configure/database-configuration-settings.rst [hunk 1]
```diff
@@ -193,6 +193,14 @@ Disable database search
 |   results.                                                    |                                                                              |
 | - **false**: **(Default)** Database search isn't disabled.    |                                                                              |
 +---------------------------------------------------------------+------------------------------------------------------------------------------+
+| Search behaviour in Mattermost changes based on which search engines are enabled.                                                            |
```
Comment (cwarnermm) [inline source/configure/database-configuration-settings.rst:196]:
```suggestion
| Search behavior in Mattermost depends on on which search engines are enabled.                                                            |
```

Comment (cwarnermm) [reply source/configure/database-configuration-settings.rst:196]: Mattermost product documentation uses US English spelling.

**Review by cwarnermm — APPROVED** [review]
Thank you, @hmmmmmmm! I've applied one small spelling correction and one phrasing update inline. My phrasing suggestion *may* impact the table formatting.

**Comment by hmmmmmmm** [comment]
Apologies, I've been away for the weekend. I'll check out the change tomorrow when I return.

File: source/configure/database-configuration-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/configure/database-configuration-settings.rst:196]: Please confirm that this text change doesn't impact the table formatting.

**Comment by hmmmmmmm** [comment]
@cwarnermm Hi, I made an update to the table formatting with the wording changes you've made.

**Review by cwarnermm — APPROVED** [review]
Thank you, @hmmmmmmm!

**Pull request merged**